js/ui/tree_list/ui.tree_list.data_controller.js

```diff
@@ -224,6 +224,16 @@ exports.DataController = dataControllerModule.controllers.data.inherit((function
             }
 
             return this._dataSource.loadDescendants(keys, childrenOnly);
+        },
+
+        /**
+         * @name dxTreeListMethods_forEachNode
+         * @publicName forEachNode(nodes, function)
+         * @param1 nodes:Array<dxTreeListNode>
+         * @param2 function:function
```

Comment:
Maybe `callback ` name will be more clear here

Reply:
@TikhonovaTatyana @RomanTsukanov ?

Reply:
`dxTreeListMethods_callback` seems to be too uncertain. `nodeProcessingCallback` or smth like ?

Reply:
Assuming that this is any JavaScript function at all, why should it be called anything other?

Reply:
Sorry, I've mixed up a bit, didn't follow you.
`callback` is the betst name suggested, no need for any more clarification. Moreover, it's a "_traditional_" name for such params, I think.

P.S. `function` could be treated as a reserved word in this case,

Reply:
If `function` does not suit
I think `callback` is a good variant